bears/general/InvalidLinkBear.py

```diff
@@ -8,15 +8,16 @@
 
 
 class InvalidLinkBear(LocalBear):
+    check_connection_url = "http://www.google.com"
```

Comment:
Have you seen this: http://stackoverflow.com/questions/3764291/checking-network-connection  ? :)
> 74.125.228.100 is one of the IP-addresses for google.com. Change http://74.125.228.100 to whatever site can be expected to respond quickly. Using a numerical IP-address avoids a DNS lookup, which may block the urllib2.urlopen call for more than a second. Thanks to rzetterberg for pointing this out.